docs/api/browser-window.md

```diff
@@ -586,6 +586,20 @@ can still bring up the menu bar by pressing the single `Alt` key.
 
 Returns whether the menu bar is visible.
 
+### BrowserWindow.setVisibleOnAllWorkspaces(visible)
+
+* `visible` Boolean
+
+Sets whether the window should be visible on all workspaces.
+
+**Note:** This API is only available on Mac/Linux.
```

Comment:
This API probably should do something on Windows 10, where Virtual Desktops are a Thing(tm)